docs/_docs/plugins.md

```diff
@@ -58,6 +58,21 @@ You have 3 options for installing plugins:
 
 <div class="note info">
   <h5>
+    The <code>:jekyll_plugins</code> group in a <code>Gemfile</code>
+  </h5>
+  <p>
+    Jekyll treats this particular group of gems(s) with a special love. Any gem
+    included in this group is given a high priority and "loaded" way before
+    Jekyll starts processing the rest of your source directory.
+    <br/>
+    Because of the priority, a gem included here will be activated (even under
+    <code>--safe</code> mode), even if its not explicitly listed under the
```

Comment:
This `--safe` note is important. It should get at least it's own sentence and perhaps even a warning box wrapper as well. Something like

```
Gems included in the `jekyll-plugins` group are activated regardless of the `--safe` mode setting. Be careful what gems are included here!
```